cypress/integration/a11y/global_variables.js

```diff
@@ -1,22 +1,28 @@
-const MOBILE = 'iphone-6';
-const DESKTOP = 'macbook-13';
-const viewports = [MOBILE, DESKTOP]
+const MOBILE = "iphone-6";
+const DESKTOP = "macbook-13";
+const viewports = [MOBILE, DESKTOP];
 
 const A11Y_CONFIG = {
-  reporter: "v2"
-}
+  reporter: "v2",
+};
 
 const EXCLUDE_CONSTANTS = [
-  ['.join-us'], // Signups
-  ['.wide-screen-menu', '.nav-links'], // Desktop Nav
-  ['.narrow-screen-menu-container', '.nav-links'], //Mobile Nav
-  ['.donate-banner *'] // Donate Banner
-  ['.site-footer', '#language-switcher'], // Language Switcher
-  ['.site-footer a.logo'] // Footer Logo
-]
+  [".join-us"], // Signups
+  [".wide-screen-menu", ".nav-links"], // Desktop Nav
+  [".narrow-screen-menu-container", ".nav-links"], //Mobile Nav
+  [".donate-banner *"][(".site-footer", "#language-switcher")], // Donate Banner // Language Switcher
+  [".site-footer a.logo"], // Footer Logo
+];
 
 const EXCLUDE_PNI_CONSTANTS = EXCLUDE_CONSTANTS.concat([
-    ['#coral-talk-stream', 'iframe'] // Commenting
-  ])
+  // ["#coral-talk-stream", "iframe"], // Commenting
```

Comment:
Do we wanna remove this part of code for good? If not, can we leave a note explaining why we decided to comment out this line instead of removing?

Reply:
I left this part in mostly in case we need to exclude commento, so that there's an example of the format we'll need to use.

Reply:
```suggestion
  // ["#coral-talk-stream", "iframe"], // leaving this in for as long as is necessary to determine whether the commento element needs excluding
```